os/pm/pm_timedsuspend.c

```diff
@@ -126,9 +125,8 @@ int pm_timedsuspend(enum pm_domain_e domain, unsigned int timer_interval)
 	pm_suspend(domain);
 	int ret = wd_start(wdog, timer_interval, (wdentry_t)timer_timeout, 2, (uint32_t)domain, (uint32_t)pid);
 	pmvdbg("PM is locked for pid %d and timer started for %d milisecond\n", pid, timer_interval);
-
-	if (ret != PM_TIMER_SUCCESS) {
-		return PM_TIMER_FAIL;
+	if (ret != OK) {
```

Comment:
I think you can remove it because it returns `ret` if you don't add some log about error.

Reply:
pm_timedsuspend is needed cleanup 
Can i change this new PR ?